crates/pallet-erc20-support/src/tests.rs

```diff
@@ -44,7 +45,29 @@ fn approve_works() {
         let approved_balance = 999;
 
         // Check test preconditions.
-        assert_eq!(Erc20Balances::approvals(alice, bob), 0);
+        assert_eq!(Erc20Balances::approvals(alice, bob), 0.into());
+
+        // Store alice-bob approval.
+        Erc20Balances::approve(alice, bob, approved_balance.into());
+
+        // Verify alice-bob approval existence.
+        assert_eq!(
+            Erc20Balances::approvals(alice, bob),
+            approved_balance.into()
+        );
+    })
+}
+
+/// This test verifies that approval logic works as expected in case approving `U256::MAX-1` value.
+#[test]
+fn approve_almost_max_works() {
```

Comment:
```suggestion
fn approve_max_works() {
```

This is in fact the maximum possible approval for the `Mock` config.